php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -35,7 +35,7 @@ abstract class NDB_BVL_Instrument extends NDB_Page
      * JSON format from the "Data" column of the flag table, or
      * from the traditional SQL tables.
      */
-    protected $jsonData = false;
+    public $jsonData = false;
```

Comment:
why was this made public ? you can reach the value from outside using `usesJSONdata()` or a soemthing like that. I dont think this variable should be modifiable by external classes

Reply:
@ridz1208 Fixed! I missed this accessor. I agree that it's safer to rely on it. Should it be the same for other properties like table, which is public?